src/makeRobot.js

```diff
@@ -39,6 +39,59 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    get info() {
+      return `name: ${this.name}, `
+        + `chip version: ${this.version}, `
+        + `wheels: ${this.wheels}`;
+    },
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goForward(steps = 1) {
+      if (steps > 0 && typeof steps === 'number') {
```

Comment:
```suggestion
      if (steps > 0) {
```

`steps > 0`  it is enough. Fix in all cases